src/main/java/hudson/plugins/promoted_builds/AddPromotionBadgeStep.java

```diff
@@ -0,0 +1,34 @@
+package hudson.plugins.promoted_builds;
```

Comment:
If it helps, I can remove this non-compilable stub and get the rest to the mergeable state. Or one can just build a new pull request on the top of it